packages/replay/test/integration/integrationSettings.test.ts

```diff
@@ -191,33 +190,28 @@ describe('Integration | integrationSettings', () => {
     it('works with default value', async () => {
       const { replay } = await mockSdk({ replayOptions: {} });
 
-      // Default is true
-      expect(replay['_recordingOptions'].maskTextSelector).toBe(MASK_ALL_TEXT_SELECTOR);
+      expect(replay['_recordingOptions'].maskAllText).toBe(true);
     });
 
     it('works with true', async () => {
       const { replay } = await mockSdk({ replayOptions: { maskAllText: true } });
 
-      expect(replay['_recordingOptions'].maskTextSelector).toBe(MASK_ALL_TEXT_SELECTOR);
+      expect(replay['_recordingOptions'].maskAllText).toBe(true);
     });
 
     it('works with false', async () => {
       const { replay } = await mockSdk({ replayOptions: { maskAllText: false } });
 
-      expect(replay['_recordingOptions'].maskTextSelector).toBe('.sentry-mask,[data-sentry-mask]');
+      expect(replay['_recordingOptions'].maskAllText).toBe(false);
     });
+  });
 
-    it('maskTextSelector takes precedence over maskAllText when not specifiying maskAllText:true', async () => {
+  describe('maskTextSelector', () => {
+    it('can have custom mask selector', async () => {
       const { replay } = await mockSdk({ replayOptions: { maskTextSelector: '[custom]' } });
 
       expect(replay['_recordingOptions'].maskTextSelector).toBe('[custom],.sentry-mask,[data-sentry-mask]');
     });
-
-    it('maskAllText takes precedence over maskTextSelector when specifiying maskAllText:true', async () => {
-      const { replay } = await mockSdk({ replayOptions: { maskAllText: true, maskTextSelector: '[custom]' } });
-
-      expect(replay['_recordingOptions'].maskTextSelector).toBe(MASK_ALL_TEXT_SELECTOR);
```

Comment:
So this fixes another bug where `maskTextSelector` was wrong when `maskAllText:true`, this should have included the default mask selectors